keyboards/smithrune/iron165r2_s/info.json

```diff
@@ -0,0 +1,133 @@
+                {"label":"G",         "matrix":[ 2, 5], "x":5.75,  "y":2          }, 
+                {"label":"H",         "matrix":[ 2, 6], "x":6.75,  "y":2          }, 
+                {"label":"J",         "matrix":[ 2, 7], "x":7.75,  "y":2          }, 
+                {"label":"K",         "matrix":[ 2, 8], "x":8.75,  "y":2          }, 
+                {"label":"L",         "matrix":[ 2, 9], "x":9.75,  "y":2          }, 
+                {"label":":",         "matrix":[ 2,10], "x":10.75, "y":2          }, 
+                {"label":"\"",        "matrix":[ 2,11], "x":11.75, "y":2          }, 
+                {"label":"~#",        "matrix":[ 2,12], "x":12.75, "y":2          }, 
+                {"label":"Enter",     "matrix":[ 2,13], "x":13.75, "y":2, "w":1.25}, 
+                {"label":"Page Down", "matrix":[ 2,14], "x":15,    "y":2          }, 
+                {"label":"Shift",     "matrix":[ 3, 0], "x":0,     "y":3, "w":1.25},
+                {"label":"\\|",       "matrix":[ 3, 1], "x":1.25,  "y":3          },
+                {"label":"Z",         "matrix":[ 3, 2], "x":2.25,  "y":3          }, 
+                {"label":"X",         "matrix":[ 3, 3], "x":3.25,  "y":3          }, 
+                {"label":"C",         "matrix":[ 3, 4], "x":4.25,  "y":3          }, 
+                {"label":"V",         "matrix":[ 3, 5], "x":5.25,  "y":3          }, 
+                {"label":"B",         "matrix":[ 3, 6], "x":6.25,  "y":3          }, 
+                {"label":"N",         "matrix":[ 3, 7], "x":7.25,  "y":3          }, 
+                {"label":"M",         "matrix":[ 3, 8], "x":8.25,  "y":3          }, 
+                {"label":"<",         "matrix":[ 3, 9], "x":9.25,  "y":3          }, 
+                {"label":">",         "matrix":[ 3,10], "x":10.25, "y":3          }, 
+                {"label":"?",         "matrix":[ 3,11], "x":11.25, "y":3          }, 
+                {"label":"Shift",     "matrix":[ 3,12], "x":12.25, "y":3, "w":1.75}, 
+                {"label":"Up",        "matrix":[ 3,13], "x":14,    "y":3          }, 
+                {"label":"End",       "matrix":[ 3,14], "x":15,    "y":3          }, 
+                {"label":"Ctrl",      "matrix":[ 4, 0], "x":0,     "y":4, "w":1.25}, 
+                {"label":"Win",       "matrix":[ 4, 1], "x":1.25,  "y":4, "w":1.25}, 
+                {"label":"Alt",       "matrix":[ 4, 2], "x":2.5,   "y":4, "w":1.25}, 
+                {"label":"Space",     "matrix":[ 4, 6], "x":3.75,  "y":4, "w":6.25}, 
+                {"label":"Alt",       "matrix":[ 4, 9], "x":10,    "y":4, "w":1.25}, 
+                {"label":"Win",       "matrix":[ 4,10], "x":11.25, "y":4, "w":1.25}, 
+                {"label":"Left",      "matrix":[ 4,11], "x":13,    "y":4          }, 
+                {"label":"Down",      "matrix":[ 4,12], "x":14,    "y":4          }, 
+                {"label":"Right",     "matrix":[ 4,13], "x":15,    "y":4          },
+                ]
+        }
+    }
+}
```

Comment:
1. formatting applied
2. "support for a myriad of alternative layouts, including a 7U bottom row and ISO", mentioned in `readme.md`, addressed
    1. LAYOUT changed to LAYOUT_all, alias added 
```suggestion
{
    "manufacturer": "SmithRune",
    "keyboard_name": "Iron165R2-S",
    "maintainer": "Gondolindrim",
    "backlight": {
        "breathing": true,
        "breathing_period": 10,
        "levels": 20,
        "pin": "A6"
    },
    "diode_direction": "COL2ROW",
    "features": {
        "audio": false,
        "backlight": true,
        "bootmagic": true,
        "command": false,
        "console": false,
        "encoder": false,
        "extrakey": true,
        "lto": true,
        "mousekey": false,
        "nkro": true,
        "rgblight": true
    },
    "indicators": {
        "caps_lock": "A4"
    },
    "matrix_pins": {
        "cols": ["A3", "B8", "B9", "B1", "B0", "B5", "B4", "B3", "A15", "A8", "A1", "A0", "C15", "C14", "C13"],
        "rows": ["B10", "A10", "A2", "A5", "A7"]
    },
    "rgblight": {
        "animations": {
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
        },
        "led_count": 22,
        "sleep": true
    },
    "url": "",
    "usb": {
        "device_version": "0.0.1",
        "pid": "0x1652",
        "vid": "0x8384"
    },
    "ws2812": {
        "pin": "B15"
    },
    "layout_aliases": {
        "LAYOUT": "LAYOUT_all"
    },
    "layouts": {
        "LAYOUT_all": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0},
                {"label": "Backspace", "matrix": [0, 14], "x": 14, "y": 0},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "|", "matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "~#", "matrix": [2, 12], "x": 12.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 13.75, "y": 2, "w": 1.25},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"label": "\\|", "matrix": [3, 1], "x": 1.25, "y": 3},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"label": "Alt", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_ansi_blocker": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "|", "matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"label": "Alt", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_ansi_blocker_split_bs": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0},
                {"label": "Backspace", "matrix": [0, 14], "x": 14, "y": 0},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "|", "matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"label": "Alt", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_ansi_blocker_tsansan": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "|", "matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                {"label": "Win", "matrix": [4, 1], "x": 1.5, "y": 4},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"label": "Space", "matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"label": "Alt", "matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_ansi_blocker_tsangan_split_bs": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0},
                {"label": "Backspace", "matrix": [0, 14], "x": 14, "y": 0},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "|", "matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                {"label": "Win", "matrix": [4, 1], "x": 1.5, "y": 4},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"label": "Space", "matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"label": "Alt", "matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_iso_blocker": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "~#", "matrix": [2, 12], "x": 12.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"label": "\\|", "matrix": [3, 1], "x": 1.25, "y": 3},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"label": "Alt", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_iso_blocker_split_bs": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0},
                {"label": "Backspace", "matrix": [0, 14], "x": 14, "y": 0},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "~#", "matrix": [2, 12], "x": 12.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"label": "\\|", "matrix": [3, 1], "x": 1.25, "y": 3},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"label": "Space", "matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"label": "Alt", "matrix": [4, 9], "x": 10, "y": 4, "w": 1.25},
                {"label": "Win", "matrix": [4, 10], "x": 11.25, "y": 4, "w": 1.25},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_iso_blocker_tsansan": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "~#", "matrix": [2, 12], "x": 12.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"label": "\\|", "matrix": [3, 1], "x": 1.25, "y": 3},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                {"label": "Win", "matrix": [4, 1], "x": 1.5, "y": 4},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"label": "Space", "matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"label": "Alt", "matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_iso_blocker_tsangan_split_bs": {
            "layout": [
                {"label": "~", "matrix": [0, 0], "x": 0, "y": 0},
                {"label": "1!", "matrix": [0, 1], "x": 1, "y": 0},
                {"label": "2@", "matrix": [0, 2], "x": 2, "y": 0},
                {"label": "3#", "matrix": [0, 3], "x": 3, "y": 0},
                {"label": "4$", "matrix": [0, 4], "x": 4, "y": 0},
                {"label": "5%", "matrix": [0, 5], "x": 5, "y": 0},
                {"label": "6^", "matrix": [0, 6], "x": 6, "y": 0},
                {"label": "7&", "matrix": [0, 7], "x": 7, "y": 0},
                {"label": "8*", "matrix": [0, 8], "x": 8, "y": 0},
                {"label": "9(", "matrix": [0, 9], "x": 9, "y": 0},
                {"label": "0)", "matrix": [0, 10], "x": 10, "y": 0},
                {"label": "-_", "matrix": [0, 11], "x": 11, "y": 0},
                {"label": "=+", "matrix": [0, 12], "x": 12, "y": 0},
                {"label": "Backspace", "matrix": [0, 13], "x": 13, "y": 0},
                {"label": "Backspace", "matrix": [0, 14], "x": 14, "y": 0},
                {"label": "Del", "matrix": [4, 14], "x": 15, "y": 0},
                {"label": "Tab", "matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"label": "Q", "matrix": [1, 1], "x": 1.5, "y": 1},
                {"label": "W", "matrix": [1, 2], "x": 2.5, "y": 1},
                {"label": "E", "matrix": [1, 3], "x": 3.5, "y": 1},
                {"label": "R", "matrix": [1, 4], "x": 4.5, "y": 1},
                {"label": "T", "matrix": [1, 5], "x": 5.5, "y": 1},
                {"label": "Y", "matrix": [1, 6], "x": 6.5, "y": 1},
                {"label": "U", "matrix": [1, 7], "x": 7.5, "y": 1},
                {"label": "I", "matrix": [1, 8], "x": 8.5, "y": 1},
                {"label": "O", "matrix": [1, 9], "x": 9.5, "y": 1},
                {"label": "P", "matrix": [1, 10], "x": 10.5, "y": 1},
                {"label": "{", "matrix": [1, 11], "x": 11.5, "y": 1},
                {"label": "}", "matrix": [1, 12], "x": 12.5, "y": 1},
                {"label": "Page Up", "matrix": [1, 14], "x": 15, "y": 1},
                {"label": "Caps Lock", "matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"label": "A", "matrix": [2, 1], "x": 1.75, "y": 2},
                {"label": "S", "matrix": [2, 2], "x": 2.75, "y": 2},
                {"label": "D", "matrix": [2, 3], "x": 3.75, "y": 2},
                {"label": "F", "matrix": [2, 4], "x": 4.75, "y": 2},
                {"label": "G", "matrix": [2, 5], "x": 5.75, "y": 2},
                {"label": "H", "matrix": [2, 6], "x": 6.75, "y": 2},
                {"label": "J", "matrix": [2, 7], "x": 7.75, "y": 2},
                {"label": "K", "matrix": [2, 8], "x": 8.75, "y": 2},
                {"label": "L", "matrix": [2, 9], "x": 9.75, "y": 2},
                {"label": ":", "matrix": [2, 10], "x": 10.75, "y": 2},
                {"label": "\"", "matrix": [2, 11], "x": 11.75, "y": 2},
                {"label": "~#", "matrix": [2, 12], "x": 12.75, "y": 2},
                {"label": "Enter", "matrix": [2, 13], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"label": "Page Down", "matrix": [2, 14], "x": 15, "y": 2},
                {"label": "Shift", "matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"label": "\\|", "matrix": [3, 1], "x": 1.25, "y": 3},
                {"label": "Z", "matrix": [3, 2], "x": 2.25, "y": 3},
                {"label": "X", "matrix": [3, 3], "x": 3.25, "y": 3},
                {"label": "C", "matrix": [3, 4], "x": 4.25, "y": 3},
                {"label": "V", "matrix": [3, 5], "x": 5.25, "y": 3},
                {"label": "B", "matrix": [3, 6], "x": 6.25, "y": 3},
                {"label": "N", "matrix": [3, 7], "x": 7.25, "y": 3},
                {"label": "M", "matrix": [3, 8], "x": 8.25, "y": 3},
                {"label": "<", "matrix": [3, 9], "x": 9.25, "y": 3},
                {"label": ">", "matrix": [3, 10], "x": 10.25, "y": 3},
                {"label": "?", "matrix": [3, 11], "x": 11.25, "y": 3},
                {"label": "Shift", "matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"label": "Up", "matrix": [3, 13], "x": 14, "y": 3},
                {"label": "End", "matrix": [3, 14], "x": 15, "y": 3},
                {"label": "Ctrl", "matrix": [4, 0], "x": 0, "y": 4, "w": 1.5},
                {"label": "Win", "matrix": [4, 1], "x": 1.5, "y": 4},
                {"label": "Alt", "matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.5},
                {"label": "Space", "matrix": [4, 6], "x": 4, "y": 4, "w": 7},
                {"label": "Alt", "matrix": [4, 10], "x": 11, "y": 4, "w": 1.5},
                {"label": "Left", "matrix": [4, 11], "x": 13, "y": 4},
                {"label": "Down", "matrix": [4, 12], "x": 14, "y": 4},
                {"label": "Right", "matrix": [4, 13], "x": 15, "y": 4}
            ]
        }
    }
}
```

Reply:
Changes added in e9fc7fd4b1 and ec9b7ddefe